Task_Array/index.js

```diff
@@ -1,67 +1,72 @@
-let arr = [1, 3, 2, 5, 4];
-
-// Filter the odd number using the filter function of the array and store it in filterArr declared below.
-
-let filterArr;
-//Write your code below
-
-//Write your code above
-console.log("Filtered Array:-", filterArr);
-
-// Find the square of each number of the array using map fumction and store it in squaredArray variable declared below. Output should be [1,9,4,16]
-let squaredArray;
-//Write your code below
-
-//Write your code above
-console.log("Squared array", squaredArray);
-
-//Remove the last element of the array arr using array functions.
-//Write your code below
-
-//Write your code above
-console.log("last element removed", arr);
-
-//Remove the first element of the array using array functions
-
-//Write your code below
-
-//Write your code above
-console.log("first element removed", arr);
```

Comment:
The arr.unshift() function does not require any arguments to remove the first element. It will remove the first element in the array without needing an argument.